src/amp/server/document.tsx

```diff
@@ -81,13 +83,19 @@ export const document = ({
     <script async custom-element="amp-live-list" src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
     <script async custom-element="amp-audio" src="https://cdn.ampproject.org/v0/amp-audio-0.1.js"></script>
 
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+
     <!-- AMP elements that are optional dependending on content -->
     ${scripts.join(' ')}
 
     <style amp-custom>
         ${getFontsCss()}
         ${resetCSS}
         ${css}
+        #reminderForm.user-invalid #reminderEmailAddress {
```

Comment:
Can we comment the reason for the break in tradition here

Reply:
What is the tradition @gtrufitt?

Reply:
Global CSS is not ideal, but I am assuming that this is because AMP adds a particular style to an element that you can't override at the component level?

Reply:
Ok, I think I've got a way of removing this. Stand by.